### ISSUE TITLE
Some blue colored content is too light.  Ensure primary color uses at least WCAG AA 4.5:1 contrast ratio

### ISSUE THREAD

**Issue by thadguidry** [OpenRefine/openrefine.org#331]
Our current Nav links and other areas of the documentation when viewed with `data-theme="light"` , the default for `root` is extremely too light and even fails AA 4.5:1 contrast ratio minimum.  (Even for me, I've noticed it with my own eyes being a bit too light).

We should add new appropriate color ranges that at least give our primary color and a few others close to the 4.5:1 contrast ratio for both `light` and `dark` theme.

We want to be at least here after changes are made, where in my example I've manually adjusted the nav bar link color.
But we'll need to do the changes in the `/src/css/custom.css` file fully :
![image](https://github.com/OpenRefine/openrefine.org/assets/986438/4efef39b-3262-4b22-aec9-20a36e4fd5ad)
